packages/graphql-serve/bin/serve.ts

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/node
+import { buildGraphbackServer, getGraphbackServerConfig } from "../src";
```

Comment:
I would prefer these files to simply call some method rather than having implementation in the bin folder

Reply:
Should I also move the arg parser stuff to another file?